icicle/tests/test_field_api.cpp

```diff
@@ -939,6 +942,128 @@ TYPED_TEST(FieldApiTest, ntt)
 }
 #endif // NTT
 
+// define program
+using MlePoly = Symbol<scalar_t>;
+MlePoly combine_func(std::vector<MlePoly>& inputs)
+{
+  const MlePoly& A = inputs[0];
+  const MlePoly& B = inputs[1];
+  const MlePoly& C = inputs[2];
+  const MlePoly& EQ = inputs[3];
+  return (EQ * (A * B - C));
+}
+
+TEST_F(FieldApiTestBase, CpuProgramExecutor)
+{
+  int seed = time(0);
```

Comment:
why not rely on the common test seed that Koren did?

Reply:
true.
Took it before Koren's push